### ISSUE TITLE
Android app claims Talk is not installed when username contains special characters

### ISSUE THREAD

**Issue by alexhorn** [nextcloud/server#34463]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments.

---

## Steps to reproduce
1. Create a user with an Umlaut (e.g. "jöhn.doe")
2. Install NextCloud Talk and NextCloud Talk for Android
3. Try to log in

### Expected behaviour
You should be able to log in.

### Actual behaviour
App claims that Talk is not installed.

## Talk app

**Talk app version:** 14.0.5

**Custom Signaling server configured:** no

**Custom TURN server configured:** no
**Custom STUN server configured:** no

## Server configuration

**Operating system**: Ubuntu 20.04

**Web server:** Apache

**Database:** MySQL

**PHP version:** 8.0

**Nextcloud Version:** 24.0.5 (via Docker)

**Comment by juliushaertl** [comment]
Right, we need the login name of course the authenticate against ldap or external storages because with an app token you would otherwise only have the userid (uuid) which cannot be used to login through LDAP/SMB for example.
